packages/lwc/types.d.ts

```diff
@@ -195,33 +195,34 @@ declare module 'lwc' {
      */
     export function wire(getType: (config?: any) => any, config?: any): PropertyDecorator;
```

Comment:
#2026 mentions an issue with the type of `@wire` decorator; im struggling to understand how the changes in this PR will fix that issue. can you explain more?

Reply:
I've updated this, and the error has gone away, but let's discuss when you have a moment?